docs/get_started/tour.mdx

```diff
@@ -45,1015 +51,828 @@ This guide is written for:
-Then call the `unreserve` function on the `TicketService`.
-
-Once you have added this to the code, try it out by calling the `UserSession/expireTicket` function:
+For example, add a ticket `seat2B` to the cart of Mary by calling the `addTicket` handler of the `CartObject`:
 
 ```shell
-curl localhost:8080/UserSession/expireTicket \
-    -H 'content-type: application/json' \
-    -d '{"key": "123", "request": "456"}'
+curl localhost:8080/CartObject/Mary/addTicket -d '"seat2B"'
 ```
 
-In the service logs, you see that the `expireTicket` function gets executed,
-after which the `unreserve` function gets executed.
-The call to `expireTicket` finishes earlier than the `unreserve` function because it didn't wait for the response of the `unreserve` function.
-
-<details>
-<summary>Show the logs</summary>
-
-```log
-[restate] [UserSession/expireTicket][inv_1gdJBtdVEcM942bjcDmb1c1khoaJe11Hbz][2024-03-18T16:14:24.671Z] DEBUG: Invoking function.
-[restate] [UserSession/expireTicket][inv_1gdJBtdVEcM942bjcDmb1c1khoaJe11Hbz][2024-03-18T16:14:24.672Z] DEBUG: Adding message to journal and sending to Restate ; BackgroundInvokeEntryMessage
-[restate] [UserSession/expireTicket][inv_1gdJBtdVEcM942bjcDmb1c1khoaJe11Hbz][2024-03-18T16:14:24.673Z] DEBUG: Journaled and sent output message ; OutputStreamEntryMessage
-//highlight-next-line
-[restate] [UserSession/expireTicket][inv_1gdJBtdVEcM942bjcDmb1c1khoaJe11Hbz][2024-03-18T16:14:24.673Z] DEBUG: Function completed successfully.
-[restate] [TicketService/unreserve][inv_1k78Krj3GqrK3GuJXkgaXBbg69R47TCeAN][2024-03-18T16:14:24.677Z] DEBUG: Invoking function.
-[restate] [TicketService/unreserve][inv_1k78Krj3GqrK3GuJXkgaXBbg69R47TCeAN][2024-03-18T16:14:24.677Z] DEBUG: Journaled and sent output message ; OutputStreamEntryMessage
-//highlight-next-line
-[restate] [TicketService/unreserve][inv_1k78Krj3GqrK3GuJXkgaXBbg69R47TCeAN][2024-03-18T16:14:24.677Z] DEBUG: Function completed successfully.
-```
-
-</details>
-
-
-</TabItem>
-<TabItem value="java" label="Java">
-
-```java
-CODE_LOAD::https://raw.githubusercontent.com/restatedev/examples/main/tutorials/tour-of-restate-java/src/main/java/dev/restate/tour/part1/UserSession.java#<start_expire_ticket>-<end_expire_ticket>
-```
-
-Create a new client by using the generated code,
-and then call `oneWay()` on the client to specify that you want to send a message without caring about the response.
-Finally, use the one-way client to call `unreserve` and supply the request.
+If this prints out `true`, then you have a working setup.
 
-Try it out by running the services via `./gradlew run` and send a request to `example.UserSession/ExpireTicket`.
+When Mary wants to proceed with the purchase, call the `checkout` handler of the `CartObject`:
 
-Call the `expireTicket` function with:
 ```shell
-curl localhost:8080/example.UserSession/ExpireTicket \
-    -H 'content-type: application/json' \
-    -d '{"user_id": "123", "ticket_id":"456"}'
+curl localhost:8080/CartObject/Mary/checkout
```

Comment:
This is a GET request i guess?

Reply:
Maybe use `POST` anyway, even if it's empty body, it's more HTTP correct and less confusing. `GET` for mutating state on a server is strange and perhaps incorrect, we might even forbid this at some point...

Reply:
Okay, I will use `-X POST`